docs/index.rst

```diff
@@ -1,117 +1,44 @@
-OpenTelemetry-Python
-====================
-
-The Python `OpenTelemetry <https://opentelemetry.io/>`_ client.
+OpenTelemetry-Python API Reference
+==================================
 
 .. image:: https://img.shields.io/badge/slack-chat-green.svg
    :target: https://cloud-native.slack.com/archives/C01PD4HUVBL
    :alt: Slack Chat
 
+Welcome to the **API docs** for the `Python OpenTelemetry client
```

Comment:
```suggestion
Welcome to the docs for the `Python OpenTelemetry implementation
```
The docs linked document more things than just the API right?

Reply:
Thanks for chiming in @cartermp.

@ocelotl - I'm ok going with your suggestion for now.

Reply:
Suggestion merged. PTAL @ocelotl et all.